parsing/parser.mly

```diff
@@ -1099,14 +1098,14 @@ rec_module_bindings:
   | rec_module_bindings and_module_binding
       { let (l, ext) = $1 in ($2 :: l, ext) }
 ;
-rec_module_binding:
+%inline rec_module_binding:
```

Comment:
Could we rename these `first_rec_module_binding` and `and_rec_module_binding` for clarity? I understand the pun with `rec` and `and`, but it doesn't work for the other instances (eg. `class_declarations`).